PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/action/CreateFunctionAppAction.java

```diff
@@ -43,19 +43,20 @@
 import com.microsoft.azuretools.utils.AzureUIRefreshCore;
 import com.microsoft.azuretools.utils.AzureUIRefreshEvent;
 import com.microsoft.intellij.util.AzureLoginHelper;
+import com.microsoft.tooling.msservices.helpers.Name;
 import com.microsoft.tooling.msservices.serviceexplorer.AzureActionEnum;
 import com.microsoft.tooling.msservices.serviceexplorer.NodeActionEvent;
 import com.microsoft.tooling.msservices.serviceexplorer.NodeActionListener;
 import com.microsoft.tooling.msservices.serviceexplorer.azure.function.FunctionModule;
-import com.microsoft.tooling.msservices.serviceexplorer.listener.ActionBasicable;
 import rx.Single;
 
 import java.util.Objects;
 import java.util.function.Consumer;
 
 import static com.microsoft.intellij.ui.messages.AzureBundle.message;
 
-public class CreateFunctionAppAction extends NodeActionListener implements ActionBasicable {
+@Name("Create")
```

Comment:
Shall we use a new annotation like `Action`? As @Name may be a bit confusing

Reply:
good idea. we can consider to do that later. but not in this scope.